content/vi/community/code-of-conduct.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Community
+layout: basic
+cid: community
+css: /css/community.css
+---
+
+<div class="community_main">
+<h1>Quy tắc ứng xử cộng đồng Kubernetes</h1>
+
+Kubernetes tuân theo
+<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">quy tắc ứng xử CNCF</a>.
+Văn bản của CNCF CoC được sao chép dưới đây, kể từ
+<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+Nếu bạn nhận thấy rằng điều này đã lỗi thời, xin vui lòng
+<a href="https://github.com/kubernetes/website/issues/new">gửi một vấn đề</a>.
+
+Nếu bạn nhận thấy vi phạm Quy tắc ứng xử tại một sự kiện hoặc cuộc họp, trong
+Slack, hoặc trong một cơ chế giao tiếp khác, hãy liên lạc với <a href="https://git.k8s.io/community/committee-code-of-conduct">Ủy ban Quy tắc ứng xử Kubernetes</a>. 
+Bạn có thể liên hệ với chúng tôi qua email tại <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
+Sự ẩn danh của bạn sẽ được bảo vệ.
+
+<div class="cncf_coc_container">
+{{< include "/static/cncf-code-of-conduct.md" >}}
```

Comment:
Hello @huynguyennovem . 
Would you look into your PR build failure:
```
0:53:51 PM: hugo --enableGitInfo --buildFuture -b https://deploy-preview-19020--kubernetes-io-master-staging.netlify.com
10:53:51 PM: Building sites …
10:54:13 PM: ERROR 2020/02/07 03:54:13 [vi] no Resource or Page matching "/static/cncf-code-of-conduct.md*".
10:54:13 PM: Total in 22038 ms
```

Reply:
@huynguyennovem please take a look on this.

Reply:
@kbhawkey I've just added missing files. Thank you for pointing that out.